backend/climateconnect_main/celery.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+from celery import Celery
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'climateconnect_main.settings')
+
+app = Celery('climateconnect_main')
+
+# Using a string here means the worker doesn't have to serialize
+# the configuration object to child processes.
+# - namespace='CELERY' means all celery-related configuration keys
+#   should have a `CELERY_` prefix.
```

Comment:
Good call, thanks for adding this. Is the string for 'climateconnect_main' or the 'django.conf:settings' bit?

Reply:
Its is for django.conf:settings -> climateconnect_main/settings.py